lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -0,0 +1,20 @@
+import { gravityLoaderWithoutAuthenticationFactory as gravityLoader } from "../api"
```

Comment:
Splits `lib/loaders/loaders_without_authentication/index.js` into three files: gravity, positron and gemini

Reply:
👍